bookwyrm/activitypub/verbs.py

```diff
@@ -231,3 +231,28 @@ class Announce(Verb):
     def action(self, allow_external_connections=True):
         """boost"""
         self.to_model(allow_external_connections=allow_external_connections)
+
+@dataclass(init=False)
+class Move(Verb):
+    """a user moving an object"""
+
+    # note the spec example for target and origin is an object but
+    # Mastodon uses a URI string and TBH this makes more sense
+    # Is there a way we can account for either format?
```

Comment:
`set_related_field` can take either an object or a string `remote_id`, which it then looks up by querying the string as a url. I'm not sure that that's directly helpful here, but it is the case that the value of a field will sometimes be a string and sometimes an actual object in a number of places.